spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/store/channel/PostgresJdbcChannelMessageStoreTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2002-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.jdbc.store.channel;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.integration.jdbc.store.PostgresJdbcMessageStore;
+import org.springframework.test.context.ContextConfiguration;
+
+/**
+ * @author Johannes Edmeier
+ */
+@ContextConfiguration
+public class PostgresJdbcChannelMessageStoreTests extends AbstractJdbcChannelMessageStoreTests {
```

Comment:
It is still can be as an extension of the `AbstractJdbcChannelMessageStoreTests`, but let's make a config based on the `PostgresContainerTest`!

Reply:
Done.